var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -208,6 +208,8 @@ class QuantumEspresso(CMakePackage):
     depends_on("blas")
     depends_on("lapack")
     depends_on("fftw-api@3")
+    depends_on("git@2.13:", type="build")
```

Comment:
What is the lowerbound based on? I'm sure it's OK, but just checking since some distro's provide older versions of git.

Reply:
The CMakeLists.txt specifies that as the minimum.

Reply:
I don't remember exactly why 2.13 was chosen. It is likely the oldest git that has been verified to work.